server/controllers/additional-sheet-parser.js

```diff
@@ -336,11 +421,11 @@ const parseTwoColumnSheet = (sheet) => {
                 }
             }
             if (sheet.data[row].length > 1) {
-                if (sheet.data[row][1] !== getSheetHeader(sheet.name, 1)) {
+                if (sheet.data[row][1] !== getSheetHeader(sheet.name, 1, row)) {
```

Comment:
Interesting what’s happening here…we either need to talk about this or if I’ve forgotten, I’ll need to refresh my memory on this game plan. It looks like `row` has been added as a parameter to `getSheetHeader`, but `getSheetHeader` only does something if `row === 0`, and as far as I can tell, all invocations of `getSheetHeader` also have `row === 0`.

It looks like you are planning some additional work here, something for this semester?